glTF/EXT_structural_metadata/PropertyAttributesPointCloud/README.md

```diff
@@ -69,19 +69,33 @@ const shaders = {
 
   // The shader that uses the `classification` value from the metadata
   // and uses this to select a color for rendering the points.
+  
+  // NOTE: The classification attribute is accessed in the vertex 
+  // shader here. When it is accessed in the fragment shader, it
+  // may not have exactly the expected value. This is tracked in
+  // https://github.com/CesiumGS/cesium/issues/10699
   CLASSIFICATION_SHADER: new Cesium.CustomShader({
-    fragmentShaderText: `
-    void fragmentMain(FragmentInput fsInput, inout czm_modelMaterial material)
-    {
+    varyings: {
+      v_color: Cesium.VaryingType.VEC3,
+    },
+    vertexShaderText: `
+    void vertexMain(VertexInput vsInput, inout czm_modelVertexOutput vsOutput) {
+        int classif = int(vsInput.attributes.classification);
```

Comment:
I was kind of sloppy when I called this `classif`... you can rename back to `classification`